user-docs/running-fakefish-on-ocp.md

```diff
@@ -0,0 +1,165 @@
+          name: fakefish-${BMC_NAME}
+          namespace: fakefish
+        spec:
+          port:
+            targetPort: http
+          tls:
+            termination: edge
+          to:
+            kind: "Service"
+            name: fakefish-${BMC_NAME}
+            weight: null
+    EOF
+    done
+    ~~~
+
+3. At this point we should have something like this:
+
+    ~~~sh
+    $ oc -n fakefish get deployment,svc,route
+
+    NAME                                     READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/fakefish-192-168-10-10   1/1     1            1           13m
+    deployment.apps/fakefish-192-168-10-20   1/1     1            1           13m
+    deployment.apps/fakefish-192-168-10-30   1/1     1            1           13m
+
+    NAME                             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+    service/fakefish-192-168-10-10   ClusterIP   172.30.108.129   <none>        9000/TCP   4m19s
+    service/fakefish-192-168-10-20   ClusterIP   172.30.205.167   <none>        9000/TCP   4m17s
+    service/fakefish-192-168-10-30   ClusterIP   172.30.171.112   <none>        9000/TCP   4m16s
+
+    NAME                                              HOST/PORT                                                           PATH   SERVICES                 PORT   TERMINATION   WILDCARD
+    route.route.openshift.io/fakefish-192-168-10-10   fakefish-192-168-10-10-fakefish.apps.mario-ipi.e2e.bos.redhat.com          fakefish-192-168-10-10   http   edge          None
+    route.route.openshift.io/fakefish-192-168-10-20   fakefish-192-168-10-20-fakefish.apps.mario-ipi.e2e.bos.redhat.com          fakefish-192-168-10-20   http   edge          None
+    route.route.openshift.io/fakefish-192-168-10-30   fakefish-192-168-10-30-fakefish.apps.mario-ipi.e2e.bos.redhat.com          fakefish-192-168-10-30   http   edge          None
+    ~~~
+
+4. And if we access one of the routes we can see that FakeFish will reply:
+
+    ~~~sh
+    $ curl -k https://fakefish-192-168-10-20-fakefish.apps.mario-ipi.e2e.bos.redhat.com/redfish/v1/
+
+    {
+        "@odata.type": "#ServiceRoot.v1_5_0.ServiceRoot",
+        "Id": "FakeFishService",
+        "Name": "FakeFish Service",
+        "RedfishVersion": "1.5.0",
+        "UUID": "not-that-production-ready",
+        "Systems": {
+            "@odata.id": "/redfish/v1/Systems"
+        },
+        "Managers": {
+            "@odata.id": "/redfish/v1/Managers"
+        },
+        "@odata.id": "/redfish/v1/",
+        "@Redfish.Copyright": "Copyright 2014-2016 Distributed Management Task Force, Inc. (DMTF). For the full DMTF copyright policy, see http://www.dmtf.org/about/policies/copyright."
+    }
+    ~~~
+
+5. Now we just need to configure our install-config or baremetalhost to use this route as the RedFish endpoint like this:
+
+    ~~~yaml
+    apiVersion: metal3.io/v1alpha1
+    kind: BareMetalHost
+    metadata:
+      name: master-0-sno
+      namespace: test-cluster
```

Comment:
Similar to below: let's use openshift-machine-api as an example to avoid dealing with assisted-specific stuff